src/__tests__/helpers.node.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @jest-environment node
```

Comment:
I'm not sure this works as expected. You probably want to pass `ssr` param to test this

Reply:
Or is it removes `jsdom`, so `window` is `undefined`? I can be wrong here